### PR TITLE
New responsible body devices homepage, and a page for requesting devices in special circumstances

### PR DESCRIPTION
**Pull request by benilovj**
### Context

Responsible body users need to know how to request devices in special circumstances

### Changes proposed in this pull request

* introduce a new 'Devices' homepage
* introduce a new 'Request devices in special circumstances' page
* update breadcrumbs and specs to match

### Guidance to review

image

image

### CODE REVIEW

File: app/views/responsible_body/devices/home/show.html.erb [hunk 1]
```diff
@@ -1,68 +1,43 @@
...
-
-    <h2 class="govuk-heading-m">A school’s full allocation can only be ordered when local coronavirus restrictions are confirmed</h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>tell us who will place orders for schools</li>
+      <li>give schools access to the services they need</li>
+      <li>see your list of schools and their allocations</li>
+    </ul>
 
-    <p class="govuk-body">We’ll contact people as soon as they’re able to place orders.</p>
+    <h2 class="govuk-heading-l govuk-!-font-size-27">
```
Comment (aldavidson) [inline app/views/responsible_body/devices/home/show.html.erb:32]: the `govuk-!-font-size-27` is redundant

**Review by aldavidson — APPROVED** [review]
LGTM, just one redundant HTML classname (not a blocker)

File: app/views/responsible_body/devices/home/show.html.erb (hunk 1 above)
Comment (fofr) [reply app/views/responsible_body/devices/home/show.html.erb:32]: It's not redundant. `27` is smaller than `l`

Comment (benilovj) [reply app/views/responsible_body/devices/home/show.html.erb:32]: Yep what @fofr said...

**Pull request merged**